lib/muchos/config/base.py

```diff
@@ -73,8 +73,8 @@
   'worker_data_dirs': None,
   'zookeeper_connect': "{% for host in groups['zookeepers'] %}{{ host }}:2181{% if not loop.last %},{% endif %}{% endfor %}",
   'zookeeper_client_port': '"2181"',
-  'zookeeper_home': '"{{ install_dir }}/zookeeper-{{ zookeeper_version }}"',
-  'zookeeper_tarball': 'zookeeper-{{ zookeeper_version }}.tar.gz',
+  'zookeeper_home': "{% if zookeeper_version.startswith('3.5') %}{{ install_dir }}/apache-zookeeper-{{ zookeeper_version }}-bin{% else %}{{ install_dir }}/zookeeper-{{ zookeeper_version }}{% endif %}",
```

Comment:
`{{ install_dir }}/` is a common prefix for both cases, so I would put it outside the conditional to simplify it slightly.

```suggestion
  'zookeeper_home': "{{ install_dir }}/{% if zookeeper_version.startswith('3.5') %}apache-zookeeper-{{ zookeeper_version }}-bin{% else %}zookeeper-{{ zookeeper_version }}{% endif %}",
```